src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
We recently had to remove ref callbacks in v3.0 (https://github.com/uber/react-map-gl/pull/310) because it will be called with null when one of the container's child components render with error. Is there a reason to set `this.container` every render pass? Shall we guard it against `null` here?